test/unit/test_matrices.py

```diff
@@ -885,11 +886,11 @@ def flush(self):
             return flush
 
         oflush = mat._flush_assembly
-        mat._flush_assembly = types.MethodType(make_flush(oflush), mat, type(mat))
+        mat._flush_assembly = types.MethodType(make_flush(oflush), mat)
```

Comment:
Should use `instancemethod` in Python 2, not necessary in Python 3, but works in both without the third argument.